demos/rocksdb_ivf/RocksDBInvertedLists.cpp

```diff
@@ -101,7 +101,8 @@ void RocksDBInvertedLists::resize(size_t /*list_no*/, size_t /*new_size*/) {
 }
 
 InvertedListsIterator* RocksDBInvertedLists::get_iterator(
-        size_t list_no) const {
+        size_t list_no,
+        void* inverted_list_contex) const {
```

Comment:
thank you for this PR, can you correct the typo please (also below)? contex -> context

Reply:
No problem~